_data/nav.yml

```diff
@@ -9,6 +9,8 @@ head:
       url: https://nbviewer.jupyter.org
       newpage: true
     - JupyterHub
+    - title: Enterprise Gateway
+      url: https://jupyter.org/enterprise_gateway
```

Comment:
this should be `url: /enterprise_gateway` like install and community above.

Reply:
I can make the change, but as the site is not local as in `community` it will only work when live, where with the current link it works both for dev (local Jekyll serving) and live website. What's the user experience we want for the website developers?

Reply:
I'd be fine w/ just posting a comment to make it obvious to future devs that the `jupyter.org` needs to be there since the `enterprise_gateway` page isn't in the jekyll site itself.

Reply:
comment added and pr updated